ManagementCenterServiceIntegrationTest.testTimedMemberState_usesCache_shortTimeFrame

http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/5641/testReport/junit/com.hazelcast.internal.management/ManagementCenterServiceIntegrationTest/testTimedMemberState_usesCache_shortTimeFrame/

```
Error Message
expected same:<{"timedMemberState":{"master":true,"time":0,"clusterName":"mc-service-tests","memberList":["127.0.0.1:5701"],"memberState":{"address":"127.0.0.1:5701","uuid":"52e402f1-dec5-48a2-840f-3945d4dea522","cpMemberUuid":null,"name":"angry_dijkstra","endpoints":[{"protocol":"MEMBER","address":{"host":"127.0.0.1","port":5701}}],"mapStats":{},"multiMapStats":{},"replicatedMapStats":{},"queueStats":{},"topicStats":{},"reliableTopicStats":{},"pnCounterStats":{},"executorStats":{},"cacheStats":{},"wanStats":{},"flakeIdStats":{},"runtimeProps":{"memory.maxMemory":1908932608,"osMemory.freePhysicalMemory":54063906816,"runtime.totalLoadedClassCount":10716,"runtime.daemonThreadCount":17,"date.startTime":1579789270654,"osSwap.freeSwapSpace":4238467072,"memory.nonHeapMemoryUsed":100658648,"os.processCpuLoad":0,"os.openFileDescriptorCount":103,"os.systemLoadAverage":77765,"runtime.loadedClassCount":10424,"osSwap.totalSwapSpace":4294963200,"os.systemCpuLoad":0,"runtime.peakThreadCount":951,"runtime.unloadedClassCount":292,"memory.totalMemory":1058537472,"seconds.upTime":152909,"memory.heapMemoryUsed":703082800,"memory.nonHeapMemoryMax":-1,"osMemory.totalPhysicalMemory":405449981952,"os.maxFileDescriptorCount":120000,"os.availableProcessors":72,"runtime.totalStartedThreadCount":4971,"runtime.availableProcessors":72,"memory.freeMemory":355454672,"memory.heapMemoryMax":1908932608,"runtime.threadCount":406,"osMemory.committedVirtualMemory":42927071232,"os.processCpuTime":114610000000},"clients":[],"beans":{"managedExecutorBeans":{"hz:scheduled":{"name":"hz:scheduled","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":144,"isTerminated":false,"completedTaskCount":1},"hz:system":{"name":"hz:system","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:offloadable":{"name":"hz:offloadable","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:query":{"name":"hz:query","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0},"hz:client":{"name":"hz:client","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:async":{"name":"hz:async","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:io":{"name":"hz:io","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0}},"eventServiceBean":{"eventThreadCount":5,"eventQueueCapacity":1000000,"eventQueueSize":0},"operationServiceBean":{"responseQueueSize":0,"operationExecutorQueueSize":0,"runningOperationsCount":0,"remoteOperationCount":0,"executedOperationCount":0,"operationThreadCount":72},"connectionManagerBean":{"clientConnectionCount":0,"activeConnectionCount":0,"connectionCount":0},"partitionServiceBean":{"partitionCount":271,"activePartitionCount":0},"proxyServiceBean":{"proxyCount":0}},"memoryStats":{"creationTime":0,"totalPhysical":405449981952,"freePhysical":54063316992,"maxNativeMemory":0,"committedNativeMemory":0,"usedNativeMemory":0,"freeNativeMemory":0,"maxHeap":1908932608,"committedHeap":1058537472,"usedHeap":703082776,"gcStats":{"creationTime":0,"minorCount":9,"minorTime":1585,"majorCount":3,"majorTime":1725,"unknownCount":0,"unknownTime":0}},"operationStats":{"maxVisibleSlowOperationCount":10,"slowOperations":[],"creationTime":1579789423266},"memberPartitionState":{"partitions":[],"memberStateSafe":true,"migrationQueueSize":0},"nodeState":{"clusterState":"ACTIVE","nodeState":"ACTIVE","clusterVersion":"4.0","memberVersion":"4.0.0","weakConfigs":{}},"hotRestartState":{"backupTaskState":"NO_TASK","backupTaskCompleted":0,"backupTaskTotal":0,"isHotBackupEnabled":false,"backupDirectory":null},"clusterHotRestartStatus":{"dataRecoveryPolicy":"FULL_RECOVERY_ONLY","hotRestartStatus":"UNKNOWN","remainingValidationTimeMillis":-1,"remainingDataLoadTimeMillis":-1,"memberHotRestartStatuses":[]},"wanSyncState":{"creationTime":0,"status":0,"syncedPartitionCount":0},"clientStats":{},"inboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}},"outboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}}},"sslEnabled":false,"lite":false,"socketInterceptorEnabled":false,"scriptingEnabled":true}}> was not:<{"timedMemberState":{"master":true,"time":0,"clusterName":"mc-service-tests","memberList":["127.0.0.1:5701"],"memberState":{"address":"127.0.0.1:5701","uuid":"52e402f1-dec5-48a2-840f-3945d4dea522","cpMemberUuid":null,"name":"angry_dijkstra","endpoints":[{"protocol":"MEMBER","address":{"host":"127.0.0.1","port":5701}}],"mapStats":{},"multiMapStats":{},"replicatedMapStats":{},"queueStats":{},"topicStats":{},"reliableTopicStats":{},"pnCounterStats":{},"executorStats":{},"cacheStats":{},"wanStats":{},"flakeIdStats":{},"runtimeProps":{"memory.maxMemory":1908932608,"osMemory.freePhysicalMemory":52236951552,"runtime.totalLoadedClassCount":10734,"runtime.daemonThreadCount":15,"date.startTime":1579789270654,"osSwap.freeSwapSpace":4238467072,"memory.nonHeapMemoryUsed":100827400,"os.processCpuLoad":0,"os.openFileDescriptorCount":100,"os.systemLoadAverage":78168,"runtime.loadedClassCount":10442,"osSwap.totalSwapSpace":4294963200,"os.systemCpuLoad":92,"runtime.peakThreadCount":951,"runtime.unloadedClassCount":292,"memory.totalMemory":1055391744,"seconds.upTime":161972,"memory.heapMemoryUsed":82535240,"memory.nonHeapMemoryMax":-1,"osMemory.totalPhysicalMemory":405449981952,"os.maxFileDescriptorCount":120000,"os.availableProcessors":72,"runtime.totalStartedThreadCount":5060,"runtime.availableProcessors":72,"memory.freeMemory":972856504,"memory.heapMemoryMax":1908932608,"runtime.threadCount":377,"osMemory.committedVirtualMemory":42982862848,"os.processCpuTime":114970000000},"clients":[],"beans":{"managedExecutorBeans":{"hz:scheduled":{"name":"hz:scheduled","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":144,"isTerminated":false,"completedTaskCount":92},"hz:system":{"name":"hz:system","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:offloadable":{"name":"hz:offloadable","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:query":{"name":"hz:query","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0},"hz:client":{"name":"hz:client","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:async":{"name":"hz:async","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:io":{"name":"hz:io","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0}},"eventServiceBean":{"eventThreadCount":5,"eventQueueCapacity":1000000,"eventQueueSize":0},"operationServiceBean":{"responseQueueSize":0,"operationExecutorQueueSize":0,"runningOperationsCount":0,"remoteOperationCount":0,"executedOperationCount":0,"operationThreadCount":72},"connectionManagerBean":{"clientConnectionCount":0,"activeConnectionCount":0,"connectionCount":0},"partitionServiceBean":{"partitionCount":271,"activePartitionCount":0},"proxyServiceBean":{"proxyCount":0}},"memoryStats":{"creationTime":0,"totalPhysical":405449981952,"freePhysical":52237492224,"maxNativeMemory":0,"committedNativeMemory":0,"usedNativeMemory":0,"freeNativeMemory":0,"maxHeap":1908932608,"committedHeap":1055391744,"usedHeap":82535240,"gcStats":{"creationTime":0,"minorCount":10,"minorTime":1701,"majorCount":3,"majorTime":1725,"unknownCount":0,"unknownTime":0}},"operationStats":{"maxVisibleSlowOperationCount":10,"slowOperations":[],"creationTime":1579789432328},"memberPartitionState":{"partitions":[],"memberStateSafe":true,"migrationQueueSize":0},"nodeState":{"clusterState":"ACTIVE","nodeState":"ACTIVE","clusterVersion":"4.0","memberVersion":"4.0.0","weakConfigs":{}},"hotRestartState":{"backupTaskState":"NO_TASK","backupTaskCompleted":0,"backupTaskTotal":0,"isHotBackupEnabled":false,"backupDirectory":null},"clusterHotRestartStatus":{"dataRecoveryPolicy":"FULL_RECOVERY_ONLY","hotRestartStatus":"UNKNOWN","remainingValidationTimeMillis":-1,"remainingDataLoadTimeMillis":-1,"memberHotRestartStatuses":[]},"wanSyncState":{"creationTime":0,"status":0,"syncedPartitionCount":0},"clientStats":{},"inboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}},"outboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}}},"sslEnabled":false,"lite":false,"socketInterceptorEnabled":false,"scriptingEnabled":true}}>
Stacktrace
java.lang.AssertionError: expected same:<{"timedMemberState":{"master":true,"time":0,"clusterName":"mc-service-tests","memberList":["127.0.0.1:5701"],"memberState":{"address":"127.0.0.1:5701","uuid":"52e402f1-dec5-48a2-840f-3945d4dea522","cpMemberUuid":null,"name":"angry_dijkstra","endpoints":[{"protocol":"MEMBER","address":{"host":"127.0.0.1","port":5701}}],"mapStats":{},"multiMapStats":{},"replicatedMapStats":{},"queueStats":{},"topicStats":{},"reliableTopicStats":{},"pnCounterStats":{},"executorStats":{},"cacheStats":{},"wanStats":{},"flakeIdStats":{},"runtimeProps":{"memory.maxMemory":1908932608,"osMemory.freePhysicalMemory":54063906816,"runtime.totalLoadedClassCount":10716,"runtime.daemonThreadCount":17,"date.startTime":1579789270654,"osSwap.freeSwapSpace":4238467072,"memory.nonHeapMemoryUsed":100658648,"os.processCpuLoad":0,"os.openFileDescriptorCount":103,"os.systemLoadAverage":77765,"runtime.loadedClassCount":10424,"osSwap.totalSwapSpace":4294963200,"os.systemCpuLoad":0,"runtime.peakThreadCount":951,"runtime.unloadedClassCount":292,"memory.totalMemory":1058537472,"seconds.upTime":152909,"memory.heapMemoryUsed":703082800,"memory.nonHeapMemoryMax":-1,"osMemory.totalPhysicalMemory":405449981952,"os.maxFileDescriptorCount":120000,"os.availableProcessors":72,"runtime.totalStartedThreadCount":4971,"runtime.availableProcessors":72,"memory.freeMemory":355454672,"memory.heapMemoryMax":1908932608,"runtime.threadCount":406,"osMemory.committedVirtualMemory":42927071232,"os.processCpuTime":114610000000},"clients":[],"beans":{"managedExecutorBeans":{"hz:scheduled":{"name":"hz:scheduled","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":144,"isTerminated":false,"completedTaskCount":1},"hz:system":{"name":"hz:system","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:offloadable":{"name":"hz:offloadable","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:query":{"name":"hz:query","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0},"hz:client":{"name":"hz:client","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:async":{"name":"hz:async","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:io":{"name":"hz:io","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0}},"eventServiceBean":{"eventThreadCount":5,"eventQueueCapacity":1000000,"eventQueueSize":0},"operationServiceBean":{"responseQueueSize":0,"operationExecutorQueueSize":0,"runningOperationsCount":0,"remoteOperationCount":0,"executedOperationCount":0,"operationThreadCount":72},"connectionManagerBean":{"clientConnectionCount":0,"activeConnectionCount":0,"connectionCount":0},"partitionServiceBean":{"partitionCount":271,"activePartitionCount":0},"proxyServiceBean":{"proxyCount":0}},"memoryStats":{"creationTime":0,"totalPhysical":405449981952,"freePhysical":54063316992,"maxNativeMemory":0,"committedNativeMemory":0,"usedNativeMemory":0,"freeNativeMemory":0,"maxHeap":1908932608,"committedHeap":1058537472,"usedHeap":703082776,"gcStats":{"creationTime":0,"minorCount":9,"minorTime":1585,"majorCount":3,"majorTime":1725,"unknownCount":0,"unknownTime":0}},"operationStats":{"maxVisibleSlowOperationCount":10,"slowOperations":[],"creationTime":1579789423266},"memberPartitionState":{"partitions":[],"memberStateSafe":true,"migrationQueueSize":0},"nodeState":{"clusterState":"ACTIVE","nodeState":"ACTIVE","clusterVersion":"4.0","memberVersion":"4.0.0","weakConfigs":{}},"hotRestartState":{"backupTaskState":"NO_TASK","backupTaskCompleted":0,"backupTaskTotal":0,"isHotBackupEnabled":false,"backupDirectory":null},"clusterHotRestartStatus":{"dataRecoveryPolicy":"FULL_RECOVERY_ONLY","hotRestartStatus":"UNKNOWN","remainingValidationTimeMillis":-1,"remainingDataLoadTimeMillis":-1,"memberHotRestartStatuses":[]},"wanSyncState":{"creationTime":0,"status":0,"syncedPartitionCount":0},"clientStats":{},"inboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}},"outboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}}},"sslEnabled":false,"lite":false,"socketInterceptorEnabled":false,"scriptingEnabled":true}}> was not:<{"timedMemberState":{"master":true,"time":0,"clusterName":"mc-service-tests","memberList":["127.0.0.1:5701"],"memberState":{"address":"127.0.0.1:5701","uuid":"52e402f1-dec5-48a2-840f-3945d4dea522","cpMemberUuid":null,"name":"angry_dijkstra","endpoints":[{"protocol":"MEMBER","address":{"host":"127.0.0.1","port":5701}}],"mapStats":{},"multiMapStats":{},"replicatedMapStats":{},"queueStats":{},"topicStats":{},"reliableTopicStats":{},"pnCounterStats":{},"executorStats":{},"cacheStats":{},"wanStats":{},"flakeIdStats":{},"runtimeProps":{"memory.maxMemory":1908932608,"osMemory.freePhysicalMemory":52236951552,"runtime.totalLoadedClassCount":10734,"runtime.daemonThreadCount":15,"date.startTime":1579789270654,"osSwap.freeSwapSpace":4238467072,"memory.nonHeapMemoryUsed":100827400,"os.processCpuLoad":0,"os.openFileDescriptorCount":100,"os.systemLoadAverage":78168,"runtime.loadedClassCount":10442,"osSwap.totalSwapSpace":4294963200,"os.systemCpuLoad":92,"runtime.peakThreadCount":951,"runtime.unloadedClassCount":292,"memory.totalMemory":1055391744,"seconds.upTime":161972,"memory.heapMemoryUsed":82535240,"memory.nonHeapMemoryMax":-1,"osMemory.totalPhysicalMemory":405449981952,"os.maxFileDescriptorCount":120000,"os.availableProcessors":72,"runtime.totalStartedThreadCount":5060,"runtime.availableProcessors":72,"memory.freeMemory":972856504,"memory.heapMemoryMax":1908932608,"runtime.threadCount":377,"osMemory.committedVirtualMemory":42982862848,"os.processCpuTime":114970000000},"clients":[],"beans":{"managedExecutorBeans":{"hz:scheduled":{"name":"hz:scheduled","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":144,"isTerminated":false,"completedTaskCount":92},"hz:system":{"name":"hz:system","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:offloadable":{"name":"hz:offloadable","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:query":{"name":"hz:query","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0},"hz:client":{"name":"hz:client","queueSize":0,"poolSize":0,"remainingQueueCapacity":7200000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:async":{"name":"hz:async","queueSize":0,"poolSize":0,"remainingQueueCapacity":100000,"maximumPoolSize":72,"isTerminated":false,"completedTaskCount":0},"hz:io":{"name":"hz:io","queueSize":0,"poolSize":0,"remainingQueueCapacity":2147483647,"maximumPoolSize":16,"isTerminated":false,"completedTaskCount":0}},"eventServiceBean":{"eventThreadCount":5,"eventQueueCapacity":1000000,"eventQueueSize":0},"operationServiceBean":{"responseQueueSize":0,"operationExecutorQueueSize":0,"runningOperationsCount":0,"remoteOperationCount":0,"executedOperationCount":0,"operationThreadCount":72},"connectionManagerBean":{"clientConnectionCount":0,"activeConnectionCount":0,"connectionCount":0},"partitionServiceBean":{"partitionCount":271,"activePartitionCount":0},"proxyServiceBean":{"proxyCount":0}},"memoryStats":{"creationTime":0,"totalPhysical":405449981952,"freePhysical":52237492224,"maxNativeMemory":0,"committedNativeMemory":0,"usedNativeMemory":0,"freeNativeMemory":0,"maxHeap":1908932608,"committedHeap":1055391744,"usedHeap":82535240,"gcStats":{"creationTime":0,"minorCount":10,"minorTime":1701,"majorCount":3,"majorTime":1725,"unknownCount":0,"unknownTime":0}},"operationStats":{"maxVisibleSlowOperationCount":10,"slowOperations":[],"creationTime":1579789432328},"memberPartitionState":{"partitions":[],"memberStateSafe":true,"migrationQueueSize":0},"nodeState":{"clusterState":"ACTIVE","nodeState":"ACTIVE","clusterVersion":"4.0","memberVersion":"4.0.0","weakConfigs":{}},"hotRestartState":{"backupTaskState":"NO_TASK","backupTaskCompleted":0,"backupTaskTotal":0,"isHotBackupEnabled":false,"backupDirectory":null},"clusterHotRestartStatus":{"dataRecoveryPolicy":"FULL_RECOVERY_ONLY","hotRestartStatus":"UNKNOWN","remainingValidationTimeMillis":-1,"remainingDataLoadTimeMillis":-1,"memberHotRestartStatuses":[]},"wanSyncState":{"creationTime":0,"status":0,"syncedPartitionCount":0},"clientStats":{},"inboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}},"outboundNetworkStats":{"bytesTransceived":{"MEMBER":0,"CLIENT":0,"WAN":0,"REST":0,"MEMCACHE":0}}},"sslEnabled":false,"lite":false,"socketInterceptorEnabled":false,"scriptingEnabled":true}}>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotSame(Assert.java:828)
	at org.junit.Assert.assertSame(Assert.java:771)
	at org.junit.Assert.assertSame(Assert.java:782)
	at com.hazelcast.internal.management.ManagementCenterServiceIntegrationTest.testTimedMemberState_usesCache_shortTimeFrame(ManagementCenterServiceIntegrationTest.java:92)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast/src/test/java/com/hazelcast/internal/management/ManagementCenterServiceIntegrationTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/management/ManagementCenterServiceIntegrationTest.java
@@ -87,9 +87,11 @@ public class ManagementCenterServiceIntegrationTest extends HazelcastTestSupport
 
     @Test
     public void testTimedMemberState_usesCache_shortTimeFrame() {
-        String responseOne = mcs.getTimedMemberStateJson().orElse(null);
-        String responseTwo = mcs.getTimedMemberStateJson().orElse(null);
-        assertSame(responseOne, responseTwo);
+        assertTrueEventually(() -> {
+            String responseOne = mcs.getTimedMemberStateJson().orElse(null);
+            String responseTwo = mcs.getTimedMemberStateJson().orElse(null);
+            assertSame(responseOne, responseTwo);
+        });
     }
 
     @Test

